### ISSUE TITLE
Two Issues: 1) Zoom-in shows content from cache.  2) Zoom-out using mouse wheel got opposite

### ISSUE THREAD

**Issue by TLKG** [wittlock/ngx-image-zoom#83]
Have a list of books, click on one will show details of the book including book cover. 

1) While lib-ngx-image loads dynamic content successfully, zoom-in always shows content of first book. For example, after reviewing and zooming-into first book from the list, start from 2nd book the magnifier content is always of the first book.
This is probably List-of-Books and Detail-of-Book are in the same component and are toggled using [hidden].

```
<!-- book detail section including book cover img -->
<div [hidden]="!Detail">
  <lib-ngx-image-zoom [thumbImage]=myCover [magnification]="1" 
            [enableScrollZoom]="true"
            [enableLens]="true"
            [lensWidth]="200">
   </lib-ngx-image-zoom>
</div>

<!-- list of book -->
<div [hidden]="Detail">
  ...
</div>
```
```
showBookDetailIncludingCover(book: any)
{
  this.Detail = true;
  this.myCover = null;  // try reset and clear, still the same.
  this.myCover = book.imgSrc;
  ...
}
```
Is it any way to reset the content of magnifier?

2) Wheeling out (zoom-out) indeed got the max zoom-in, then zoom-out.

Thank you.

**Comment by mstojevski** [comment]
You should also provide fullImage even its same as thumbImage, so both thumbImage and magnifer updates.
`<lib-ngx-image-zoom [thumbImage]=myCover [magnification]="1" 
            [enableScrollZoom]="true"
             [fullImage]="myCover" // Add this
            [enableLens]="true"
            [lensWidth]="200">
   </lib-ngx-image-zoom>`

**Comment by TLKG** [comment]
Correct, that's how I found out the fix. Thank you very much for the response.

On May 16, 2021 at 10:20 AM, Milan Stojevski ***@***.***> wrote:






You should also provide fullImage even its same as thumbImage, so both thumbImage and magnifer updates.
<lib-ngx-image-zoom [thumbImage]=myCover [magnification]="1" [enableScrollZoom]="true" [fullImage]="myCover" // Add this [enableLens]="true" [lensWidth]="200"> </lib-ngx-image-zoom>
